src/Components/test/E2ETest/Tests/WebAssemblyAuthenticationTests.cs

```diff
@@ -60,6 +60,7 @@ static WebAssemblyAuthenticationTests()
         protected override void InitializeAsyncCore()
         {
             Navigate("/", noReload: true);
+            Browser.Manage().Window.Size = new Size(1024, 800);
```

Comment:
This is "orthogonal" but it was causing a bug in the Auth tests, there's a separate issue tracking the fix for Identity UI